recipes/xerces-c/all/conanfile.py

```diff
@@ -14,6 +14,7 @@ class XercesCConan(ConanFile):
     description = (
         "Xerces-C++ is a validating XML parser written in a portable subset of C++"
     )
+    version = "3.2.5"
```

Comment:
```suggestion
```
Please, do not specify the version in the recipe. It should be passed by command line, so it will be consumed from conandata.yml